arch/x86_64/mcount-dynamic.c

```diff
@@ -623,6 +623,7 @@ int mcount_unpatch_func(struct mcount_dynamic_info *mdi, struct uftrace_symbol *
 
 	switch (mdi->type) {
 	case DYNAMIC_FENTRY:
+	case DYNAMIC_PATCHABLE:
```

Comment:
But there's nothing to unpatch for `-fpatchable-function-entry` as they are already NOPs.

Reply:
It does its job in the following example.
```
$ g++ -fpatchable-function-entry=5 example.cpp

$ uftrace record -P. -U ^std:: a.out
```
Since `-P.` asks patching all the available functions, we need a way to suppress the previous patching request.

Reply:
Ok, multiple updates also make it useful.